sympy/plotting/plot.py

```diff
@@ -2289,6 +2358,9 @@ def plot3d_parametric_surface(*args, show=True, **kwargs):
     series = []
     plot_expr = check_arguments(args, 3, 2)
     series = [ParametricSurfaceSeries(*arg, **kwargs) for arg in plot_expr]
+    xlabel = kwargs.setdefault("xlabel", "x")
+    ylabel = kwargs.setdefault("ylabel", "y")
+    kwargs.setdefault("zlabel", "f(%s, %s)" % (xlabel, ylabel))
```

Comment:
For parametric line and surface, I think that it should better stay as `z`

Reply:
I agree. Done it.